src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -54,7 +61,7 @@ private static void BuildTocMap(Context context, Document fileToBuild, TableOfCo
                 Debug.Assert(tocMapBuilder != null);
                 Debug.Assert(fileToBuild != null);
 
-                var (errors, tocModel, _, referencedDocuments, referencedTocs) = Load(context, fileToBuild);
+                var (errors, tocModel, _, referencedDocuments, referencedTocs, _) = Load(context, fileToBuild);
```

Comment:
We don't care about the `errors` anymore?

Reply:
it is `monikers`, not `errors`.
when build the toc map, we don't need to care about the monikers

---
In reply to: [235277827](https://github.com/dotnet/docfx/pull/3655#discussion_r235277827) [](ancestors = 235277827)

Reply:
Since the `Load` method will be invoked twice, we can ignore the `errors` here

---
In reply to: [235283050](https://github.com/dotnet/docfx/pull/3655#discussion_r235283050) [](ancestors = 235283050,235277827)